db/seeds.rb

```diff
@@ -1,7 +1,173 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+puts "Cleaning Database 🧼"
+sleep 1
+print " ."
+sleep 1
+print " ."
+sleep 1
+puts " ."
+sleep 1
+puts ""
+
+# Clear database to prevent duplicate records
+Friendship.destroy_all
+User.destroy_all
+
+# Create users
+emma = User.create(
+  username: "emmvs", 
+  email: "emma@test.com", 
+  password: "123456", 
+  first_name: "Emma", 
+  middle_name: "Anna Agneta", 
+  last_name: "Rünzel",
+  nickname: "Emmsi",
+  emoji: "☀️"
+)
+
+josh = User.create(
+  username: "xxjmsxx", 
+  email: "josh@test.com", 
+  password: "123456", 
+  first_name: "Joshua", 
+  middle_name: "Matthew", 
+  last_name: "Smith",
+  nickname: "Josh",
+  emoji: "🍰"
+)
+
+santi = User.create(
+  username: "santiagosan93", 
+  email: "santi@test.com", 
+  password: "123456", 
+  first_name: "Santiago", 
+  middle_name: nil, 
+  last_name: "Sanchez",
+  nickname: "Santi",
+  emoji: "🐘"
+)
+
+puts "Created #{User.count} Users 🤷🏼‍♂️🧑🏽‍🦱💁🏼‍♀️"
```

Comment:
Not needed for this PR. But in case we don't know if we created 1 or more users, we could go with [`#pluralize`](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) here.

```suggestion
puts "Created #{User.count} #{pluralize(User.count, "user")} 🤷🏼‍♂️🧑🏽‍🦱💁🏼‍♀️"
```